lib/cc/engine/coffeelint_results.rb

```diff
@@ -1,8 +1,7 @@
 module CC
   module Engine
     class CoffeelintResults
-      def initialize(directory, config: nil)
-        @directory = directory
+      def initialize(config: nil)
```

Comment:
I know you didn't introduce this, but is there value in this being a key parameter, instead of `config = nil`?